app/components/Vault/index.js

```diff
@@ -540,54 +539,54 @@ const Vault = (props) => {
           <tbody>
             <tr>
               <td>veCRV APY</td>
-              <td>{truncateApy(apy.data.poolApy)}</td>
+              <td>{truncateApy(apy.composite.pool_apy)}</td>
             </tr>
             <tr>
               <td>Boost</td>
               <td>{currentBoost}x</td>
             </tr>
             <tr>
               <td>Total APY</td>
-              <td>{truncateApy(apy.data.totalApy)}</td>
+              <td>{truncateApy(apy.composite.pool_apy)}</td>
             </tr>
           </tbody>
         </TooltipTable>
       </div>
     );
-  } else if (apyType === 'curve') {
-    const currentBoost = _.get(apy, 'data.currentBoost', 0).toFixed(2);
+  } else if (apyType === 'crv') {
+    const currentBoost = _.get(apy, 'composite.boost', 0).toFixed(2);
     apyTooltip = (
       <div>
-        {apy.description}
+        Pool APY + Boosted CRV APY
         <br />
         <br />
         <TooltipTable>
           <tbody>
             <tr>
               <td>Pool APY</td>
-              <td>{truncateApy(apy.data.poolApy)}</td>
+              <td>{truncateApy(apy.composite.pool_apy)}</td>
             </tr>
-            {apy.data.tokenRewardsApr > 0 && (
+            {apy.composite.rewards_apr > 0 && (
               <tr>
                 <td>Bonus Rewards APR</td>
-                <td>{truncateApy(apy.data.tokenRewardsApr)}</td>
+                <td>{truncateApy(apy.composite.rewards_apr)}</td>
               </tr>
             )}
             <tr>
               <td>Base CRV APR</td>
-              <td>{truncateApy(apy.data.baseApr)}</td>
+              <td>{truncateApy(apy.composite.base_apr)}</td>
             </tr>
             <tr>
               <td>Boost</td>
               <td>{currentBoost}x</td>
             </tr>
             <tr>
               <td>Total APY</td>
-              <td>{truncateApy(apy.data.totalApy)}</td>
+              <td>{truncateApy(apy.gross_apr)}</td>
```

Comment:
is this ok @x48-crypto ? In the new PR we use 
`totalApy: newApy.composite.pool_apy ? newApy.composite.pool_apy : newApy.composite.totalApy`